pkg/kubelet/dockertools/manager.go

```diff
@@ -245,6 +252,21 @@ func NewDockerManager(
 		optf(dm)
 	}
 
+	apiVersion, err := dm.APIVersion()
+	if err != nil {
+		glog.Errorf("Failed to get api version from docker %v", err)
+	}
+
+	daemonVersion, err := dm.Version()
+	if err != nil {
+		glog.Errorf("Failed to get daemon version from docker %v", err)
+	}
+
+	// Update version cache periodically
+	go wait.Until(func() {
+		dm.versionCache.Update(dm.machineInfo.MachineID, apiVersion, daemonVersion)
+	}, 5*time.Second, wait.NeverStop)
```

Comment:
`5 seconds` seems like too often. I recommend changing it to a minute at the least.